natlas-server/app/ipscanmanager.py

```diff
@@ -0,0 +1,58 @@
+from netaddr import *
+from app.cyclicprng import *
```

Comment:
I really don't like `import *` if I can help it, because you lose the namespace of the items you're importing, making it more likely to have conflicting names. Can you import explicitly the things you need?